utils/intellisensePrepublish.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+
+const generateTokensArray = require('./generateTokensArray');
+
+// if the file is being run directly
+const generateTokensForPrepublish = () => {
+  const outDir = '../polaris-intellisense/src/data/allTokens.ts';
+
+  const tokens = generateTokensArray(outDir);
```

Comment:
```suggestion
  const tokens = generateTokensArray();
```

`generateTokensArray` currently doesn't accept any input